.github/workflows/python.yml

```diff
@@ -36,34 +36,39 @@ jobs:
 
           # OSX py builds
           - name: osx 3.6 intel
-            os: macos-latest
+            os: macos-13
             python: "3.6"
             piparch: macosx_10_9_intel
 
           - name: osx 3.7 intel
-            os: macos-latest
+            os: macos-13
             python: "3.7"
             piparch: macosx_10_9_intel
 
-          - name: osx 3.8 intel
+          - name: osx 3.8 arm64
             os: macos-latest
             python: "3.8"
-            piparch: macosx_10_9_intel
+            piparch: macosx_11_0_arm64
```

Comment:
I changed these to `arm64` as the `macos-latest` is now only `arm64` by default. In order to build for `x86_64` it would need to build `macos-latest-large`
https://github.com/actions/runner-images?tab=readme-ov-file#available-images

I will look into `_universal2` however